src/components/ListUsers.vue

```diff
@@ -11,7 +11,7 @@
             <div class="bg-gray-100 px-5 py-2 rounded flex flex-column gap-3">
               <div class="flex flex-wrap align-items-center gap-2 justify-content-between">
                 <div class="uppercase font-light font-sm text-gray-400 mb-1">
-                  {{ singularizeFirestoreCollection(orgType) }}
+                  {{ i18n.t('listUsers.districtTitle') }}
```

Comment:
This should be a specific Audience type. All you need to do here is make sure 'district' renders as 'Site'.